### PR TITLE
PW-7745- Move away from access token 

### PR DESCRIPTION
**Pull request by raoulritter**
**Description**

This PR updates the webapi.xml following magento's best practices for v9. 

**Tested scenarios**

Fixes

### CODE REVIEW

File: Helper/PaymentResponseHandler.php [hunk 1]
```diff
@@ -105,6 +105,7 @@ public function formatPaymentResponse($resultCode, $action = null, $additionalDa
                 return [
                     "isFinal" => true,
                     "resultCode" => $resultCode,
+                    "donationToken" => $donationToken
```
Comment (candemiralp) [inline Helper/PaymentResponseHandler.php:108]: `donationToken` parameter should be include in the response for only `AUTHORISED` and `RECEIVED` event codes.

File: Model/AdyenInitiateTerminalApi.php [hunk 2]
```diff
@@ -114,7 +114,7 @@ public function initiate($payload)
         // Decode payload from frontend
         $payload = json_decode($payload, true);
 
-        // Validate JSON that has just been parsed if it was in a valid format
+        //Validate JSON that has just been parsed if it was in a valid format
         if (json_last_error() !== JSON_ERROR_NONE) {
```
Comment (peterojo) [inline Model/AdyenInitiateTerminalApi.php:118]: Could you please undo this change? seems unnecessary

File: Model/Api/AdyenOrderPaymentStatus.php [hunk 3]
```diff
@@ -78,4 +78,4 @@ public function getOrderPaymentStatus(string $orderId): string
             !empty($additionalInformation['additionalData']) ? $additionalInformation['additionalData'] : null
         ));
     }
-}
```
Comment (peterojo) [inline Model/Api/AdyenOrderPaymentStatus.php:81]: Please undo this as well, there should be a new line at the end of the file

File: Helper/PaymentResponseHandler.php [hunk 4]
```diff
@@ -100,11 +100,15 @@ public function formatPaymentResponse($resultCode, $action = null, $additionalDa
     {
         switch ($resultCode) {
             case self::AUTHORISED:
+                return [
+                    "isFinal" => true,
+                    "resultCode" => $resultCode,
```
Comment (candemiralp) [inline Helper/PaymentResponseHandler.php:105]:
```suggestion
                    "resultCode" => $resultCode
```

File: Helper/PaymentResponseHandler.php [hunk 5]
```diff
@@ -125,7 +129,7 @@ public function formatPaymentResponse($resultCode, $action = null, $additionalDa
                 return [
                     "isFinal" => true,
                     "resultCode" => $resultCode,
-                    "additionalData" => $additionalData
+                    "additionalData" => $additionalData,
```
Comment (candemiralp) [inline Helper/PaymentResponseHandler.php:132]:
```suggestion
                    "additionalData" => $additionalData
```

**Review by candemiralp — DISMISSED** [review]

**Review by peterojo — APPROVED** [review]

**Review by candemiralp — DISMISSED** [review]

**Review by candemiralp — APPROVED** [review]

**Review by peterojo — APPROVED** [review]
